playbooks/remove_controller_services/tasks/main.yml

```diff
@@ -54,6 +54,20 @@
       service: name={{ item }} state=stopped enabled=false must_exist=false
       with_items: "{{ service_list }}"
       failed_when: false
+    - name: remove neutron agent conf files
+      file: "/etc/init/{{ service_list }}.conf" state=absent
```

Comment:
Yep put this in on my way out the door, will fix first thing in the morning, thanks

Will actually be able to run it in dev tomorrow too, woo!